test/e2e/common/pods.go

```diff
@@ -282,16 +226,6 @@ var _ = framework.KubeDescribe("Pods", func() {
 					{
 						Name:  "nginx",
 						Image: "gcr.io/google_containers/nginx-slim:0.7",
-						Ports: []api.ContainerPort{{ContainerPort: 80}},
-						LivenessProbe: &api.Probe{
-							Handler: api.Handler{
-								HTTPGet: &api.HTTPGetAction{
-									Path: "/index.html",
-									Port: intstr.FromInt(8080),
-								},
-							},
-							InitialDelaySeconds: 30,
-						},
```

Comment:
Because this test doesn't need a liveness check, so remove dependency here to avoid the feature affecting irrelevant test. :)